cli/src/main/java/keywhiz/cli/configs/DescribeActionConfig.java

```diff
@@ -28,4 +28,7 @@ public class DescribeActionConfig {
 
   @Parameter(names = "--name", description = "Name of the item to describe", required = true)
   public String name;
+
+  @Parameter(names = { "-d", "--include-deleted" }, description = "Whether to include any deleted secret(s) that match the provided name")
+  public Boolean includeDeleted = false;
```

Comment:
Since we are providing a non-null default, I believe this should be a lowercase-b boolean (there's precedent in the existing config classes for this style). And while technically we wouldn't need to specify a default in that case, that is the existing practice, so I'd stick with it.

Reply:
sounds good. looks like I need to learn what the difference between `Boolean` and `boolean` is in Java